python/ray/tune/tests/test_trial_runner.py

```diff
@@ -376,7 +376,7 @@ def train(config, reporter):
             }
         })
         self.assertEqual(trial.status, Trial.TERMINATED)
-        self.assertEqual(trial.last_result[TIMESTEPS_TOTAL], 100)
+        self.assertEqual(trial.last_result[DONE], True)
```

Comment:
`self.assertEqual(trial.last_result[TIMESTEPS_TOTAL], 100)` should also be true right?

Reply:
I believe so. Though that behavior is already check for in the cleaned up `testReportTimeStep`.

There is a bit of overlap between `testReportTimeStep`, `testAbruptReturn`, and `testNoDoneReceived`. We could probably combine `testAbruptReturn` and `testNoDoneReceived`, and leave `testReportTimeStep` to check for the timestep counting behavior. What do you think?

Reply:
Yeah I think that's fair - can you combine those two?

Reply:
Done